src/blocks/homepage-articles/edit.tsx

```diff
@@ -434,107 +398,163 @@ class Edit extends Component< HomepageArticlesProps > {
+							disabled={ true }
+						/>
 					) : (
 						! specificMode && (
 							<>
 								<ToggleControl
-									label={ __( 'Show "Load more posts" Button', 'newspack-blocks' ) }
+									label={ __( 'Show "Load more posts" button', 'newspack-blocks' ) }
 									checked={ moreButton }
 									onChange={ () => setAttributes( { moreButton: ! moreButton } ) }
 								/>
-								{ moreButton && (
-									<ToggleControl
-										label={ __( 'Infinite Scroll', 'newspack-blocks' ) }
-										checked={ infiniteScroll }
-										onChange={ () => setAttributes( { infiniteScroll: ! infiniteScroll } ) }
-									/>
-								) }
+								<ToggleControl
+									label={ __( 'Infinite scroll', 'newspack-blocks' ) }
+									checked={ infiniteScroll }
+									disabled={ ! moreButton }
+									onChange={ () => setAttributes( { infiniteScroll: ! infiniteScroll } ) }
+								/>
 							</>
 						)
 					) }
-					<ToggleControl
-						label={ __( 'Allow duplicate stories', 'newspack-blocks' ) }
-						help={ __(
-							"If checked, this block will be excluded from the page's de-duplication logic. Duplicate stories may appear.",
-							'newspack-blocks'
-						) }
-						checked={ ! attributes.deduplicate }
-						onChange={ ( value: boolean ) => setAttributes( { deduplicate: ! value } ) }
-						className="newspack-blocks-deduplication-toggle"
-					/>
 				</PanelBody>
-				<PanelBody title={ __( 'Featured Image Settings', 'newspack-blocks' ) }>
-					<PanelRow>
+				<PanelBody title={ __( 'Display', 'newspack-blocks' ) } className="newspack-block__panel">
+					<BaseControl
+						label={ __( 'Text', 'newspack-blocks' ) }
+						id="newspack-block__content-display"
+						className="newspack-block__button-group"
+					>
+						<ButtonGroup>
+							<Button
+								variant={ ! showExcerpt && ! showFullContent && 'primary' }
+								aria-pressed={ ! showExcerpt && ! showFullContent }
+								onClick={ () => {
+									setAttributes( {
+										showExcerpt: false,
+										showFullContent: false
+									} )
+								} }
+							>
+								{ __( 'None', 'newspack-blocks' ) }
+							</Button>
+							<Button
+								variant={ showExcerpt && ! showFullContent && 'primary' }
+								aria-pressed={ showExcerpt && ! showFullContent }
+								onClick={ () => {
+									setAttributes( {
+										showExcerpt: ! showExcerpt,
+										showFullContent: showFullContent ? false : showFullContent
+									} )
+								} }
+							>
+								{ __( 'Excerpt', 'newspack-blocks' ) }
+							</Button>
+							<Button
+								variant={ ! showExcerpt && showFullContent && 'primary' }
+								aria-pressed={ ! showExcerpt && showFullContent }
+								onClick={ () => {
+									setAttributes( {
+										showFullContent: ! showFullContent,
+										showExcerpt: showExcerpt ? false : showExcerpt
+									} )
+								} }
+							>
+								{ __( 'Full Post', 'newspack-blocks' ) }
```

Comment:
Non-blocking suggestion:

```suggestion
								{ __( 'Full Content', 'newspack-blocks' ) }
```

Reply:
Unfortunately, "Content" is too long.
![Screenshot 2025-02-06 at 13 58 56@2x](https://github.com/user-attachments/assets/8060247d-94f0-4417-86b5-84fa89701fe5)

Reply:
Unless I remove "Full"?

![Screenshot 2025-02-06 at 14 33 50@2x](https://github.com/user-attachments/assets/d71f4cf5-ca9c-4a56-9ea4-e69e980eff1b)

Reply:
I'm okay with keeping it "Full post" since it's clearer to me than just "Content"